src/Lykke.Service.PayInternal.Core/Domain/PaymentRequests/PaymentRequestStatus.cs

```diff
@@ -43,6 +43,21 @@ public enum PaymentRequestStatus
         /// <summary>
         /// An error occurred during processing payment request.
         /// </summary>
-        Error
+        Error,
+
+        /// <summary>
+        /// Payment request is accepted to settlement.
+        /// </summary>
+        SettlementInProgress,
```

Comment:
Are these new status being mapped to Invoice statuses?

Reply:
LPDEV-900 and LPDEV-901 are created.